hardware/eVehicles/eVehicle.cpp

```diff
@@ -34,6 +36,7 @@ License: Public domain
 #define VEHICLE_ALERT_LOCK 2
 #define VEHICLE_LEVEL_BATTERY 1
 #define VEHICLE_COUNTER_ODO 1
+#define VEHICLE_CUSTOM 9
```

Comment:
You add two custom types (custom sensor and custom text). I would make two definitions to separate the different device types (e.g. VEHICLE_CUSTOM_SENSOR and VEHICLE_CUSTOM_TEXT.

Reply:
Hm.. I interpreted the defines as referring to what they measure/mean and not the the type of Domoticz sensor.

As these defines are used for the ID of the domoticz sensors (see also the screenshot I put on the forum topic for this PR) all custom sensors can be easily identified as they all share the same (parent) ID.

I was actually looking for a way to let the user decide if they want custom sensors to be created at all or not.